halium-install

```diff
@@ -38,6 +38,35 @@ do_shell()
 	adb shell "$@"
 }
 
+do_chroot()
+{
+	do_shell "chroot /cache/system $@"
+}
+
+check_busybox()
+{
+    export device_architecture=$(do_shell "uname -m")
+    export busybox_exists=$(do_shell "if [ -d /sbin/busybox ]; then echo true; else echo false; fi")
+    mkdir busybox
+
+    if [ $busybox_exists = "true" ]; then
+        echo 'Using already installed busybox'
+    else
+        if [ $device_architecture = x86_64 ]; then
+            wget https://github.com/meefik/busybox/raw/master/app/src/main/assets/intel/static/bin/busybox -O busybox/busybox
```

Comment:
We can use the armv6l variant from https://busybox.net/downloads/binaries/1.26.2-defconfig-multiarch/ . It's working just fine on armv7